### PR TITLE
KEYCLOAK-12729 Add password policy not-email

### PR DESCRIPTION
**Pull request by thomasdarimont**
Added test cases and initial translations

### CODE REVIEW

**Comment by thomasdarimont** [comment]
The tests work from me locally - perhaps I'm missing a cleanup procedure after the test run.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/RegisterTest.java [hunk 1]
```diff
@@ -491,6 +491,28 @@ public void registerUserNotUsernamePasswordPolicy() {
         assertEquals("Please specify username.", registerPage.getError());
     }
 
+    // KEYCLOAK-12729
+    @Test
+    public void registerUserNotEmailPasswordPolicy() {
+
+        RealmRepresentation notEmailRealm = RealmBuilder.create().passwordPolicy("notEmail").build();
+        notEmailRealm.setRegistrationEmailAsUsername(true);
```
Comment (mposolda) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/RegisterTest.java:499]: @thomasdarimont As you pointed, you are missing cleanup of those things after the test method.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/account/AccountFormServiceTest.java [hunk 2]
```diff
@@ -412,6 +413,23 @@ public void changePasswordWithNotUsernamePolicy() {
         events.expectAccount(EventType.UPDATE_PASSWORD).assertEvent();
     }
 
+    @Test
+    public void changePasswordWithNotEmailPolicy() {
+        setPasswordPolicy("notEmail(1)");
```
Comment (mposolda) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/account/AccountFormServiceTest.java:418]: I guess this probably worth cleanup as well.

**Review by mposolda — CHANGES_REQUESTED** [review]
@thomasdarimont Thanks for the contribution! It looks good to me. Just 2 very minor things:
- Cleanup will be probably needed in some tests as you already pointed in some comment above
- It will be good to send small PR to the documentation as well to make sure this page is updated with the new policy https://www.keycloak.org/docs/latest/server_admin/index.html#_password-policies

**Comment by mposolda** [comment]
@thomasdarimont Any update regarding this PR?

**Comment by mposolda** [comment]
@thomasdarimont Hi, any update on this PR or are you in the middle of PTO season? :-)

**Review by mposolda — APPROVED** [review]
@thomasdarimont Cool, Thanks! :-)

**Pull request merged**